stdlib/config-api/src/main/ballerina/src/config/natives.bal

```
@@ -14,16 +14,66 @@
+
 # Checks whether the given key is in the configuration registry.
 #
 # + key - The configuration key to be looked-up
 # + return - Returns true if the key is present; if not returs false
-public function contains(@untainted string key) returns boolean = external;
+public function contains(@untainted string key) returns boolean {
+    return externContains(java:fromString(key));
+}
+
+function externContains(handle key) returns boolean = @java:Method {
+    name: "contains",
+    class: "org.ballerinalang.stdlib.config.Contains"
+} external;
 
 # Sets the specified key/value pair as a configuration.
 #
 # + key - The key of the configuration value to be set
 # + value - The configuration value to be set
-public function setConfig(string key, string|int|float|boolean value) = external;
+public function setConfig(string key, string|int|float|boolean value) {
+    if (value is string) {
+        externSetConfig(java:fromString(key), java:fromString(value));
+    } else {
+         externSetConfig(java:fromString(key), value);
+    }
+
+}
+
+function externSetConfig(handle key, handle|int|float|boolean value) = @java:Method {
+    name: "setConfig",
+    class: "org.ballerinalang.stdlib.config.SetConfig"
+} external;
+
+public function get(@untainted string key, ValueType vType) returns string|int|float|boolean|map<any> {
+    string valueType = "ARRAY";
+    if (vType is STRING) {
+        valueType = "STRING";
+    } else if (vType is INT) {
+        valueType = "INT";
+    } else if (vType is FLOAT) {
+        valueType = "FLOAT";
+    } else if (vType is MAP) {
+        valueType = "MAP";
+    } else if (vType is BOOLEAN) {
+        valueType = "BOOLEAN";
+    }
+    var result = externGet(java:fromString(key), java:fromString(valueType));
+    string value = "";
+    if (result is handle) {
+        var stringResult = java:toString(result);
+        if (stringResult is string) {
+            value = stringResult;
+        }
+        return value;
```

Shouldn't we return an error here ?